lua/session_manager/init.lua

```diff
@@ -13,13 +13,26 @@ function session_manager.load_last_session(bang)
   end
 end
 
+function session_manager.load_current_session(bang)
+  local cwd = vim.fn['getcwd']()
+  local session_name = utils.dir_to_session_filename(cwd)
+  local tmp_f = io.open(session_name, "r")
+  if tmp_f ~= nil then
```

Comment:
Add `local Path = require('plenary.path')` after the line 2 and use the following:

```suggestion
  if Path:new(session_name):exists() ~= nil then
```

Reply:
This suggestion can't be applied automatically, you need to add the mentioned `require`.

Reply:
done

Reply:
Thanks!